installer/uninstall-cloudflow.sh

```diff
@@ -25,7 +25,7 @@ fi
 
 
 # shellcheck source=common/detect.sh
-source "$currentDirectory"/detect.sh
+. common/utils.sh
```

Comment:
This should be detect.sh

```suggestion
. common/detect.sh
```

Reply:
Updated 👍